src/notation/view/notationpaintview.cpp

```diff
@@ -761,6 +783,15 @@ PointF NotationPaintView::toLogical(const QPoint& point) const
     return PointF::fromQPointF(m_matrix.inverted().map(scaledPoint.toQPoint()));
 }
 
+PointF NotationPaintView::fromLogical(const QPoint& point) const
+{
+    double scale = guiScaling();
```

Comment:
This line might better placed a few lines lower, close the `return` statement where `scale` is actually used.